### DISCUSSION TITLE
Inportation of image metadata from process not reflecting accurate sampler.

### DISCUSSION THREAD

**Discussion by djwashout** [vladmandic/sdnext#3186] [General]
When I drop a DPM++SDE image in to the process tab and then send data to text it will display DPM++SDE in sampler, however images produced are UniPC.

![Screenshot 2024-05-30 205518](https://github.com/vladmandic/automatic/assets/132242585/32458a10-1941-4f5c-b58d-d3cd2c48ff71)
![Screenshot 2024-05-30 205607](https://github.com/vladmandic/automatic/assets/132242585/9a6c710d-f6cc-418e-b76a-2bb03eb10325)

**Comment by vladmandic** [comment]
pls upload the image used to reproduce this.

**Reply by djwashout** [reply]
NSFW okay?

**Comment by djwashout** [comment]
This was produced on Google Colab, Version: app=sd.next updated=2023-11-03 hash=dc79cec1
Trying to change the sampler to DPM ++3M EDM produces static. Other samplers produce a similar but not exact image. 
 
![20240524030236_490052401_henmixReal_v40_DPM++ SDE_45_1024_1536_ A1111](https://github.com/vladmandic/automatic/assets/132242585/e3b954b9-630d-40aa-b5e7-71bcbc102f54)

**Comment by djwashout** [comment]
So this would happen occasionally when i would drop images over form A1111 in to Vlad, id have to make sure to select the right sampler in the drop down, however i don't see the option for DPM++SDE so i can not take corrective action..

**Comment by vladmandic** [comment]
i've noticed an issue with negative prompt parsing, that is now fixed.

the image was generated using sdnext with backend=original and now you're using backend=diffusers which has different schedulers.
scheduler `dpm++ sde` does not exist as such as neither **dpm sde** can have **++ solver **nor **dpm++** can have **sde solver**.
closest you can get is to set **dpm++ 1s** with **sde** solver.

note that solver is set in settings -> schedulers and it modifies behavior of currently selected scheduler.

**Reply by djwashout** [reply]
Wow thanks! I noticed sometimes when I would load an image into process and send it to text it will not load the negative prompt and i would just cut and paste it in myself.

Also i have changed back to backend=original and i now have DPM++SDE for sd1.5 models. I am now able to reproduce the image.
Sorry for my lack of understanding regarding samplers, schedulers, and solvers. i'm trying to read up on it all.

Thanks Vlad. You do have a far superior product here. It has more creativity and produces amazing images. I have always preferred it over a1111.
